src/components/Map.js

```diff
@@ -6,20 +6,25 @@ import Pin from './Pin';
 import UserLocationPin from './UserLocationPin';
 import DialogBody from './DialogBody';
 import { MapCenterContext } from '../context/MapCenterContext';
+import { GoogleMapsContext } from '../context/GoogleMapsContext';
 
 export const Map = ({
   defaultCenterMap,
   centerMap,
   locations = [],
-  zoom = 16,
+  zoom = 14,
```

Comment:
Just randomly curious why this was reduced from 16 to 14! 😄 That's all!

Reply:
There's a conversation with Alejandro and the mentors on this in our pair thread in slack.  I think it's to more likely show result pin on the map if you don't live in a large city like Paris etc and/or in a more rural area.

Reply:
I like your curiosity, Jessica 😄 👍 and thank you, Megan, for your good answer! Only a small comment about this change: there are some places or address where we can't see many pins (or nothing) around them immediately but if you zoom out a little you can see more pins. Of course, it depends on what place or address you want to search. As Megan mentioned we have a thread where you can more details about this change 😄 .